weather_sp/splitter_pipeline/file_splitters.py

```diff
@@ -24,7 +24,9 @@
 import numpy as np
 import pygrib
 import xarray as xr
+from apache_beam.io.filesystem import DEFAULT_READ_BUFFER_SIZE
 from apache_beam.io.filesystems import FileSystems
+from apache_beam.io.gcp.gcsio import WRITE_CHUNK_SIZE
```

Comment:
Can you add a comment or TODO or something that we don't want to bring in more gcs-specific dependencies and instead should clean this up?

Reply:
ack